### ISSUE TITLE
Cannot delete nested translations when using `--fix`

### ISSUE THREAD

**Issue by kcarmonamurphy** [simplabs/ember-intl-analyzer#441]
Thanks for creating a really neat tool. I was using the `--fix` option and this addon threw the following error:

```
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at deleteNestedTranslation (/Users/kcmurphy/.asdf/installs/nodejs/14.16.1/.npm/lib/node_modules/ember-intl-analyzer/index.js:343:14)
    at deleteNestedTranslation (/Users/kcmurphy/.asdf/installs/nodejs/14.16.1/.npm/lib/node_modules/ember-intl-analyzer/index.js:348:7)
    at deleteNestedTranslation (/Users/kcmurphy/.asdf/installs/nodejs/14.16.1/.npm/lib/node_modules/ember-intl-analyzer/index.js:348:7)
    at /Users/kcmurphy/.asdf/installs/nodejs/14.16.1/.npm/lib/node_modules/ember-intl-analyzer/index.js:317:9
    at Array.forEach (<anonymous>)
    at removeUnusedTranslations (/Users/kcmurphy/.asdf/installs/nodejs/14.16.1/.npm/lib/node_modules/ember-intl-analyzer/index.js:313:26)
    at run (/Users/kcmurphy/.asdf/installs/nodejs/14.16.1/.npm/lib/node_modules/ember-intl-analyzer/index.js:83:5)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

We have lots of nested translations in different files in the following format:
```
key1:
  key2:
    key3: 'translation'
  key4: 'another translation'
...
```

and our folder structure is nested as well:
```
app > translations > folder1 > folder2 > en-ca.yml
app > translations > folder1 > folder2 > fr-ca.yml

app > translations > folder1 > folder3 > en-ca.yml
app > translations > folder1 > folder3 > fr-ca.yml
...
```

**Comment by Mikek2252** [comment]
@kcarmonamurphy Hey thanks for the issue and info, I have managed to track down the bug and written a fix. I'm creating a MR now and will update this issue when it has been reviewed and released.

**Comment by kcarmonamurphy** [comment]
excellent, thank you @Mikek2252!

**Comment by Mikek2252** [comment]
This bug should now be fixed in the 3.1.1 release, sorry for the delay. Im going to close this issue now, feel free to reopen if you are still experiencing it.
